src/algorithm.rs

```diff
@@ -834,6 +931,40 @@ impl Bft {
                     self.new_round_start();
                 }
             }
+
+            #[cfg(feature = "verify_req")]
+            BftMsg::VerifyResp(verify_resp) => {
+                self.save_verify_resp(verify_resp);
+                if self.step == Step::VerifyWait {
+                    // next do precommit
+                    self.change_to_step(Step::Precommit);
+                    if self.check_verify() == VERIFY_UNDETERMINED {
```

Comment:
How could this happen?

Reply:
Receive a `VerifyResp` message while the node is at `Step::VerifyWait` .